source/deployment/mobile-app-deployment.rst

```diff
@@ -15,67 +15,57 @@ Leaders, Administrators, and/or Champions
     **Note:** Some features described in this guide are only available in Enterprise Edition E10 and/or E20.
 
 *******************
-Learning Objectives
+Learning objectives
 *******************
 
 This guide provides foundational information necessary when developing a plan for a proof of concept and/or enterprise mobile deployment. 
 
-It does not include technical implementation guidance. Links to relevant documentation are included however.
```

Comment:
Recommend moving to the end of this section and modifying to "Technical implementation guides are included as  links to keep the information in this guide focused."